### PR TITLE
Develop2

### PR DESCRIPTION
**Pull request by dminyaylo**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: list,
+                      cleaning_staff: object
```
Comment (Vasyl-Smutok) [inline app/cinema/hall.py:8]:
```suggestion
                      customers: list[???],
                      cleaning_staff: Cleaner

```

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

File: app/cinema/hall.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: list[???],
```
Comment (ArsenPidhoretskyi) [inline app/cinema/hall.py:7]: list[???]
What is this???
Provide correct inner type
What you expect inside your function

File: app/cinema/bar.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class CinemaBar:
+    @staticmethod
+    def sell_product(product: str, customer: object) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/cinema/bar.py:3]: customer: object
Specify more correct type

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
```
Comment (ArsenPidhoretskyi) [inline app/main.py:7]: customers: list of what?
Provide inner type

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**